interface/modules/custom_modules/oe-module-faxsms/WARP.md

```diff
@@ -0,0 +1,674 @@
+# WARP.md - OpenEMR FaxSMS Module
```

Comment:
I'm confused about WARP.md being in a module that's part of core. Shouldn't the modules be shipped as separate repos and installable via packagist, and then individualized agents instructions makes sense?